### ISSUE TITLE
Incorrect window decoration on Linux

### ISSUE THREAD

**Issue by Rapiz1** [ilime/Petal#31]
* System info: archlinux 
* Steps to reproduce: 
1. Start Petal
2. Hover cursor on the close button. It's the same problem with minimalize button.
* Terminal Output:
```
-------------------------------------
Node version:  10.11.0
Electron version:  4.2.5
Chrome version:  69.0.3497.128
-------------------------------------
db file existed.
```
* Screenshot
![image](https://user-images.githubusercontent.com/20105061/61546234-bb34a680-aa7b-11e9-87c1-d87a3707f74a.png)
BTW, the cursor is hidden in the above screenshot. So the popup tip is as expected.

**Comment by Rapiz1** [comment]
[Fixed](https://github.com/Rapiz1/Petal/blob/dev/src/styles/Sidebar.scss)

**Comment by g1eny0ung** [comment]
Fixes in #33. Very thankful.
